homeassistant/helpers/update_coordinator.py

```diff
@@ -162,14 +157,21 @@ async def async_refresh(self) -> None:
         await self._async_refresh(log_failures=True)
 
     async def _async_refresh(
-        self, log_failures: bool = True, raise_on_auth_failed: bool = False
+        self,
+        log_failures: bool = True,
+        raise_on_auth_failed: bool = False,
+        scheduled: bool = False,
     ) -> None:
         """Refresh data."""
         if self._unsub_refresh:
             self._unsub_refresh()
             self._unsub_refresh = None
 
         self._debounced_refresh.async_cancel()
+
+        if scheduled and self.hass.is_stopping:
```

Comment:
Why check is scheduled?

Reply:
We still need to allow manual refresh after the stop event per https://github.com/home-assistant/core/pull/39172#discussion_r475271555